benchmarks/benchmark/tools/profile-generator/container/benchmark_serving.py

```diff
@@ -134,6 +134,90 @@ def init_errors_map() -> Dict[str, int]:
   }
   return errors
 
+async def send_stream_request(
+    backend: str,
+    api_url: str,
+    prompt: str,
+    prompt_len: int,
+    output_len: int,
+    best_of: int,
+    use_beam_search: bool,
+    top_k: int,
+    tokenizer: PreTrainedTokenizerBase,
+    sax_model: str,
+    model: str,
+) -> Tuple[Tuple[int, int, float], float, Dict[str, int]]:
+  """Sends stream request to server"""
+  request_start_time = time.time()
+  errors = init_errors_map()
+
+  headers = {"User-Agent": "Benchmark Client"}
+  # Doesn't work with if model weights are stored in PD on the model server side
```

Comment:
Why is this the case?

Reply:
Oops forgot to remove this comment. This was before https://github.com/GoogleCloudPlatform/ai-on-gke/commit/a3401f29d0110a8d62dccd6ff1f5217f7766b1a3 was merged. Before, tokenizer was used for both the tokenizer and the model. So if the model was downloaded in a PD and stored in a directory not called for instance `meta-llama/Llama-3.1-405B-Instruct `,  the model server would not find the model.